packages/e2e-tests/register/models/SetRecoveryOption.js

```diff
@@ -0,0 +1,55 @@
+class SetRecoveryOptionPage {
+    recoveryOptions = {
+        ledger: "ledger",
+        email: "email",
+        phone: "phone",
+        seedPhrase: "phrase",
+    };
+    constructor(page) {
+        this.page = page;
+    }
+    async navigate(accountId) {
+        await this.page.goto(`/set-recovery/${accountId}`);
+    }
+    // optionName is option prop in code (currently /phrase|ledger|email|phone/)
+    getLedgerSelector() {
+        return `data-test-id=recoveryOption.${this.recoveryOptions.ledger}`;
+    }
+    getEmailSelector() {
+        return `data-test-id=recoveryOption.${this.recoveryOptions.email}`;
+    }
+    getPhoneSelector() {
+        return `data-test-id=recoveryOption.${this.recoveryOptions.phone}`;
+    }
+    getSeedPhraseSelector() {
+        return `data-test-id=recoveryOption.${this.recoveryOptions.seedPhrase}`;
+    }
+    async clickLedgerRecoveryOption() {
+        return await this.page.click(this.getLedgerSelector());
+    }
+    async clickEmailRecoveryOption() {
+        return await this.page.click(this.getEmailSelector());
+    }
+    async clickPhoneRecoveryOption() {
+        return await this.page.click(this.getPhoneSelector());
+    }
+    async clickSeedPhraseRecoveryOption() {
+        return await this.page.click(this.getSeedPhraseSelector());
+    }
+    async getLedgerRecoveryOptionElementHandle() {
+        return await this.page.$(this.getLedgerSelector());
+    }
+    async getEmailRecoveryOptionElementHandle() {
+        return await this.page.$(this.getEmailSelector());
+    }
+    async getPhoneRecoveryElementHandle() {
+        return await this.page.$(this.getPhoneSelector());
+    }
+    async getSeedPhraseRecoveryElementHandle() {
+        return await this.page.$(this.getSeedPhraseSelector());
+    }
+    async submitRecoveryOption() {
+        await this.page.click(`[type="submit"]`);
```

Comment:
Would prefer if this was clicking on an element with an explicit ID; otherwise this really isn't `submitRecoveryOption()`, it's just a generic 'click form submit button' action and doesn't really have any value as part of the page model, and it leaves the possibility of us ending up in a weird state where the UI isn't where we expect it to be, but there is a submit there (even if it's the _wrong_ submit button for a totally different form), so we click it because the selector here is so generic.  It can make it very difficult to figure out why the test failed because tapping on the wrong `submit` button will likely cause a navigation even farther away from the screen where we initially got 'lost' in the flow.  We want the test to  throw / fail _as close as possible_ to when the state of our UI is inconsistent with what we expect and being explicit here helps us towards that goal even if the suite that uses this function isn't _extremely_ explicit about asserting after each step.